spring-boot/pom.xml

```diff
@@ -58,15 +58,15 @@
 		<dependency>
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
-			<version>42.2.23</version>
+			<version>42.5.0</version>
```

Comment:
did this change on its own when you updated sentry-spring-boot and sentry-logback?

or was this part of an Upgrade Pathway per the sentry sdk instructions?

@Angelodaniel

Reply:
This was due to a command for Maven which retrieved the latest packages.
When upgrading the dependencies these got upgraded as well 
https://stackoverflow.com/questions/27798691/maven-force-update-only-for-specific-dependency-mvn-u-for-selected-dependencie

Reply:
ok great